const init = async () => {
    try {
        
    } catch (err) {
        console.log(err);
    }
}

export default init